newVersion.py

```diff
@@ -259,59 +335,87 @@ def all_command(args, dirs):
+        epilog=textwrap.dedent('''\
+            Order of operations
+            -------------------
+            1. newVersion.py bb
+            2. Release iTwin/mobile-sdk-ios and npm publish @itwin/mobile-core
+            3. newVersion.py bumpui
+            4. npm publish @itwin/mobile-ui-react
+            5. newVersion.py bumpsamples
+            '''))
     sub_parsers = parser.add_subparsers(title='Commands', metavar='')
-    
-    parser_bump = sub_parsers.add_parser('bump', help='Create new point release')
-    parser_bump.set_defaults(func=bump_command)
-    parser_bump.add_argument('-n', '--new', dest='new_version', help='New release version')
-    parser_bump.add_argument('-f', '--force', action=argparse.BooleanOptionalAction, dest='force', help='Force even if local changes already exist')
+
+    parser_commit = sub_parsers.add_parser('branch', help='Branch changes')
+    parser_commit.set_defaults(func=branch_command)
+    parser_commit.add_argument('-n', '--new', dest='new_mobile', help='New iTwin Mobile SDK release version')
+    parser_commit.add_argument('-f', '--force', action=argparse.BooleanOptionalAction, dest='force', help='Force even if local changes don\'t exist in all directories')
 
     parser_change = sub_parsers.add_parser('change', help='Change version (alternative to bump, specify versions)')
     parser_change.set_defaults(func=change_command)
-    parser_change.add_argument('-n', '--new', dest='new_version', help='New release version', required=True)
-    parser_change.add_argument('-nb', '--newITwin', dest='new_itwin', help='New @itwin package version', required=True)
-    parser_change.add_argument('-ni', '--newIos', dest='new_ios', help='New itwin-mobile-native-ios version', required=True)
+    parser_change.add_argument('-n', '--new', dest='new_mobile', help='New iTwin Mobile SDK release version', required=True)
+    parser_change.add_argument('-ni', '--newITwin', dest='new_itwin', help='New @itwin package version', required=True)
+    parser_change.add_argument('-na', '--newAddOn', dest='new_add_on', help='New itwin-mobile-native-ios version', required=True)
     parser_change.add_argument('-f', '--force', action=argparse.BooleanOptionalAction, dest='force', help='Force even if local changes already exist')
 
-    parser_commit = sub_parsers.add_parser('branch', help='Branch changes')
-    parser_commit.set_defaults(func=branch_command)
-    parser_commit.add_argument('-n', '--new', dest='new_version', help='New release version')
+    parser_bump = sub_parsers.add_parser('bump', help='Create new point release')
+    parser_bump.set_defaults(func=bump_command)
+    parser_bump.add_argument('-n', '--new', dest='new_mobile', help='New iTwin Mobile SDK release version')
+    parser_bump.add_argument('-f', '--force', action=argparse.BooleanOptionalAction, dest='force', help='Force even if local changes already exist')
+
+    parser_bump = sub_parsers.add_parser('bb', help='Execute bump then branch')
```

Comment:
Since `bb` is often used for BentleyBuild, I think it would be better to use a different command name here to avoid confusion.  Maybe `bumpbranch`?

Reply:
Done.